src/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { range, reorder, kOrderKeys } from '../../../../../common/util/util.js';
+import { kShaderStageCombinationsWithStage } from '../../../../capability_info.js';
+
+import {
+  kLimitBaseParams,
+  makeLimitTestGroup,
+  kBindGroupTests,
+  kBindingCombinations,
+  getPipelineTypeForBindingCombination,
+  getPipelineAsyncTypeForBindingCombination,
+  getPerStageWGSLForBindingCombination,
+} from './limit_utils.js';
+
+const limit = 'maxSampledTexturesPerShaderStage';
+export const { g, description } = makeLimitTestGroup(limit);
+
+g.test('createBindGroupLayout,at_over')
```

Comment:
Does the spec also validate the limits at pipeline layout creation? I don't know that this is tested at the moment.

Reply:
Yes, good catch! Will add test

Reply:
added